doxygen/dox/Specifications.dox

```diff
@@ -20,4 +23,7 @@
 \li \ref TBLSPEC
 \li \ref sec_dim_scales_spec
 
+<hr>
+Navigate back: \ref index
```

Comment:
Is this one missing "Main", or is it just special?

Reply:
They are pairs, it should match the top one